index.ts

```diff
@@ -0,0 +1,64 @@
+import { IncomingMessage, ServerResponse } from 'http';
+import depd from 'depd';
+
+const deprecate = depd('hsts');
```

Comment:
Why is this being double-imported?

Reply:
This matches the previous behaviour:

```js
var deprecate = require('depd')('hsts')
```

Reply:
Ah, apologies—I misread. Resolving this thread.